CHANGELOG.md

```diff
@@ -4,6 +4,26 @@
 
 ### Internal
 
+- [In Progress: 22 Sep 2021] Custom Drug Entry Sheet UI Improvements
+- [In Progress: 31 Aug 2021] Refactor logic around providing drug frequencies label depending on the country
+- [In Progress: 2 Sep 2021] Add a progress state in `CustomDrugEntrySheet`
```

Comment:
> Add a progress state in `CustomDrugEntrySheet`

this is already completed and already in the demo changelog. Can we remove this from here? There must have been some mistake while rebasing somewhere which is why this is here.

Reply:
I am guessing the drug entry sheet UI improvements is blocked by the final PR that is open @janhavisinghh ?

Reply:
Yeah, I'm talking about the progress state changelog

Reply:
Yeah yeah, I am editing the changelog. So checking to see if I should remove the in-progress for the above entry as well.